pkg/virtctl/guestfs/guestfs.go

```diff
@@ -379,7 +377,7 @@ func (client *K8sClient) getPodsForPVC(pvcName, ns string) ([]corev1.Pod, error)
 	return pods, nil
 }
 
-func setFSGroupLibguestfs() (*int64, error) {
+func setFSGroupLibguestfs(root bool, fsGroup string) (*int64, error) {
```

Comment:
Have you considered changing this and other funcs to be receiving funcs of `guestfsCommand` instead of passing args down the chain?

Reply:
i did ye, wasnt sure if it fits
i can try

Reply:
I'd still change this and the other set funcs to be receiving funcs of `guestfsCommand`.